cocos/2d/CCNode.h

```diff
@@ -38,6 +38,10 @@
 #include "2d/CCComponentContainer.h"
 #include "2d/CCComponent.h"
 
+#ifdef CC_STUDIO_ENABLED_VIEW   // for cocostudio only
+#include "CocosStudioExtension.h"
+#endif
+
```

Comment:
Many component inherit from node use "ResouceData" define, add it to here so we won't need add this header file to every component use it.